TRB_label_assist/clustering.py

```diff
@@ -53,6 +71,9 @@ def add_loc_clusters(
         Args:
             loc_df (dataframe): must have columns 'start_lat' and 'start_lon' 
                 or 'end_lat' and 'end_lon'
+            loc_entry ( list of Entry/confirmedTrip): list consisting all entries from the
+                time data was loaded. loc_df was obtained from this by converting to df and 
+                then filtering out labeled trips and expanding user_inputs   
```

Comment:
nit: this is inconsistent - if it is a list of confirmed trip entries, then you might want to rename it `trip_entry` instead of `loc_entry`. `loc` generally stands for `location`.

Reply:
fixed.